Dockerfile

```diff
@@ -33,6 +33,8 @@ COPY docker/nginx.conf /etc/netbox-nginx/nginx.conf
 
 WORKDIR /opt/netbox/netbox
 
+RUN pip3 install djangorestframework==3.6.4
```

Comment:
What is the reason you didn't put this right after Line 28?

Reply:
Also, in the code above we always used `pip` instead of `pip3` (see line 28 and line 18).

I would like to keep the possibility open for also building this image for python 2.7. This could be done by just changing the `FROM` part on line 1. But it would not work when we use `pip3` explicitly somewhere.

Reply:
And possibly add a comment with a short explanation of this and a link to the issue in `digitalocean/netbox`, so that we can remove this workaround once they fixed it on their side.